crowbar_framework/app/assets/stylesheets/content/content.scss

```diff
@@ -45,21 +45,24 @@
   }
 }
 
-.dropdown-menu {
-  border-radius: 6px;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-  right: -1px;
+h1,
+h2,
+h3.modal-title {
+  color: #2a8d8e;
```

Comment:
Color literals like `#2a8d8e` should only be used in variable declarations; they should be referred to via variable everywhere else.